### PR TITLE
Add information about immediate save

### PR DESCRIPTION
**Pull request by dominik-zeglen**
I want to merge this change because it adds information that any change to attributes in product types will result in immediate save.

**PR intended to be tested with API branch:** main

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-6058 Finilize button disabled in product type editor](https://app.clickup.com/t/2549495/SALEOR-6058)

File: src/config.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { ListSettings, ListViews, Pagination } from "./types";
 
 export const APP_MOUNT_URI = process.env.APP_MOUNT_URI || "/";
 export const APP_DEFAULT_URI = "/";
-export const API_URI = process.env.API_URI;
+export const API_URI = "http://localhost:8000/graphql/";
```
Comment (Droniu) [inline src/config.ts:7]: Why is this changed?

File: src/components/AssignContainerDialog/AssignContainerDialog.tsx [hunk 2]
```diff
@@ -95,14 +92,14 @@ const AssignContainerDialog: React.FC<AssignContainerDialogProps> = props => {
       fullWidth
       maxWidth="sm"
     >
-      <DialogTitle>{title}</DialogTitle>
+      <DialogTitle>{labels.title}</DialogTitle>
```
Comment (Droniu) [inline src/components/AssignContainerDialog/AssignContainerDialog.tsx:95]: Shouldn't this go through intl?

Comment (dominik-zeglen) [reply src/components/AssignContainerDialog/AssignContainerDialog.tsx:95]: Well it's easier to pass translated string rather than `MessageDescriptor`, so it's done in the ancestor component

**Review by Droniu — APPROVED** [review]

**Review by bmigirl — APPROVED** [review]
